projects/dsp-ui/src/lib/viewer/values/text-value/text-value-as-xml/text-value-as-xml.component.ts

```diff
@@ -90,12 +91,14 @@ export class TextValueAsXMLComponent extends BaseValueComponent implements OnIni
             toolbar: ['heading', '|', 'bold', 'italic', 'link', 'bulletedList', 'numberedList', 'blockQuote', 'underline', 'strikethrough', 'subscript', 'superscript', 'horizontalline', 'insertTable', 'code', 'codeBlock', 'removeformat', 'redo', 'undo'],
             heading: {
                 options: [
-                    {model: 'heading1', view: 'h1', title: 'Heading 1'},
-                    {model: 'heading2', view: 'h2', title: 'Heading 2'},
-                    {model: 'heading3', view: 'h3', title: 'Heading 3'},
-                    {model: 'heading4', view: 'h4', title: 'Heading 4'},
-                    {model: 'heading5', view: 'h5', title: 'Heading 5'},
-                    {model: 'heading6', view: 'h6', title: 'Heading 6'},
+                    { model: 'paragraph', title: 'Paragraph', class: 'ck-heading_paragraph' },
+                    { model: 'heading1', view: 'h1', title: 'Heading 1' },
+                    { model: 'heading2', view: 'h2', title: 'Heading 2' },
+                    { model: 'heading3', view: 'h3', title: 'Heading 3' },
+                    { model: 'heading4', view: 'h4', title: 'Heading 4' },
+                    { model: 'heading5', view: 'h5', title: 'Heading 5' },
+                    { model: 'heading6', view: 'h6', title: 'Heading 6' },
+                    { model: 'formatted', view: 'pre', title: 'Formatted' }
```

Comment:
@loicjaouen I could add support for the `<pre>` element. However, according to the docs it is not available in [CKeditor 5](https://ckeditor.com/docs/ckeditor5/latest/builds/guides/migrate.html#configuration-options-compatibility-table):

![Screenshot 2020-10-23 at 13 42 02](https://user-images.githubusercontent.com/6000023/96999523-90b28280-1535-11eb-807d-d6c75b25f56a.png)

Reply:
I posted the question on stackoverflow: https://stackoverflow.com/questions/64499528/support-of-element-pre-in-ckeditor-5